libopenarc/arc-internal.h

```diff
@@ -122,4 +122,13 @@ typedef struct arc_plist ARC_PLIST;
 struct arc_canon;
 typedef struct arc_canon ARC_CANON;
 
+/*
+**  memory allocation wrappers
+*/
+
+#define ARC_malloc  malloc
```

Comment:
I think macros should be all-caps.  The code looks good but I'd like to be consistent with current use throughout the code base.  Do you mind making that change and resubmitting one more time?